tests/jenkins/jenkinsjob-regression-files/packer/packer-build.jenkinsfile.txt

```diff
@@ -10,14 +10,10 @@
             packer-build.script(groovy.lang.Closure)
                packer-build.checkout({$class=GitSCM, branches=[{name=main}], userRemoteConfigs=[{url=https://github.com/opensearch-project/opensearch-ci}]})
                packer-build.readJSON({file=./packer/jenkins-agent-al2-arm64.json})
-               packer-build.string({credentialsId=jenkins-agent-vpc-id, variable=vpc_id})
-               packer-build.string({credentialsId=jenkins-agent-subnet-id, variable=subnet_id})
-               packer-build.string({credentialsId=jenkins-agent-sg-id, variable=sg_id})
-               packer-build.withCredentials([vpc_id, subnet_id, sg_id], groovy.lang.Closure)
-               packer-build.writeJSON({file=./packer/substitute_jenkins-agent-al2-arm64.json, json={variables={name-base=Jenkins-Agent-AL2-X64, os-version=AL2, build-region=us-east-1, build-vpc=vpc_id, build-subnet=subnet_id, build-secgrp=sg_id, build-time={{isotime "2006-01-02T03-04-05Z"}}, aws_ami_region=us-east-1}}, pretty=2})
-               packer-build.string({credentialsId=jenkins-aws-account-public, variable=AWS_ACCOUNT_PUBLIC})
-               packer-build.withCredentials([AWS_ACCOUNT_PUBLIC], groovy.lang.Closure)
-                  packer-build.withAWS({role=opensearch-packer, roleAccount=AWS_ACCOUNT_PUBLIC, duration=3600, roleSessionName=jenkins-session, useNode=true}, groovy.lang.Closure)
+               packer-build.withSecrets({secrets=[{envVar=vpc_id, secretRef=op://opensearch-infra-secrets/packer-build-ids/jenkins-agent-vpc-id}, {envVar=subnet_id, secretRef=op://opensearch-infra-secrets/packer-build-ids/jenkins-agent-subnet-id}, {envVar=sg_id, secretRef=op://opensearch-infra-secrets/packer-build-ids/jenkins-agent-sg-id}]}, groovy.lang.Closure)
+               packer-build.writeJSON({file=./packer/substitute_jenkins-agent-al2-arm64.json, json={variables={name-base=Jenkins-Agent-AL2-X64, os-version=AL2, build-region=us-east-1, build-vpc=vpc_id_123, build-subnet=subnet_id_123, build-secgrp=sg_id_123, build-time={{isotime "2006-01-02T03-04-05Z"}}, aws_ami_region=us-east-1}}, pretty=2})
```

Comment:
Will the secrets value be shown in the logging or this is just testing bug?

Reply:
There is no secret showing here?
It is just the name of the secret not the secret values.